components/size_aware_image.jsx

```diff
@@ -47,13 +49,19 @@ export default class SizeAwareImage extends React.PureComponent {
          * css classes that can added to the img as well as parent div on svg for placeholder
          */
         className: PropTypes.string,
+
+        /*
+         * Enables the logic of surrounding small images with a bigger container div for better click/tap targeting
+         */
+        handleSmallImageContainer: PropTypes.bool,
     }
 
     constructor(props) {
         super(props);
 
         this.state = {
             loaded: false,
+            isSmallImage: false,
```

Comment:
I believe this will cause issue if we assume all images are small when we load container because we would want to provide enough space for this before hand. If we only set the right value after image loads we cause the content to realign. Can we assign this based on the available dimensions if it exists on load as well?

Reply:
Sure! 👍